src/main/java/com/gluonhq/substrate/model/ReleaseConfiguration.java

```diff
@@ -27,16 +27,18 @@
  */
 package com.gluonhq.substrate.model;
 
+import java.util.Objects;
+
 public class ReleaseConfiguration {
 
     public static final String DEFAULT_BUNDLE_VERSION = "1.0";
```

Comment:
Can you order these defaults, like we have for all/Mac/iOS/Android platforms below?